test/mock/createInstantSearch.ts

```diff
@@ -7,7 +7,7 @@ import defer from '../../src/lib/utils/defer';
 export const createInstantSearch = (
   args: Partial<InstantSearch> = {}
 ): InstantSearch => {
-  const searchClient = createSearchClient();
+  const searchClient = args.client || createSearchClient();
```

Comment:
```suggestion
  const searchClient = args.searchClient || createSearchClient();
```

Reply:
although it's also possible to do this with default in the destructuring, like in the following line

Reply:
but `searchClient` doesn't exist in `args`.
I've updated it to be destructured from `args` though: https://github.com/algolia/instantsearch.js/pull/4464/commits/8300a1db7678224ab1509013da817d8500e6cc2e